src/containers/Secrets/Secrets.js

```diff
@@ -154,11 +187,39 @@ export /* istanbul ignore next */ class Secrets extends Component {
           />
         )}
         <Table
-          handleNew={this.handleNewSecretClick}
-          handleDelete={this.handleDeleteSecretClick}
+          title="Secrets"
+          headers={initialHeaders}
+          rows={secretsFormatted}
           loading={loading}
-          secrets={secrets}
           selectedNamespace={selectedNamespace}
+          emptyTextAllNamespaces={intl.formatMessage(
+            {
+              id: 'dashboard.emptyState.allNamespaces',
+              defaultMessage: 'No {kind} under any namespace.'
+            },
+            { kind: 'Secrets' }
+          )}
+          emptyTextSelectedNamespace={intl.formatMessage(
+            {
+              id: 'dashboard.emptyState.selectedNamespace',
+              defaultMessage: 'No {kind} under namespace {selectedNamespace}'
+            },
+            { kind: 'Secrets', selectedNamespace }
+          )}
+          batchActionButtons={[
+            {
+              onClick: this.handleDeleteSecretClick,
+              text: 'Delete',
```

Comment:
need to use `intl` for these user-facing strings

Reply:
updated.